DOCUMENTATION.md

```diff
@@ -730,16 +732,16 @@ compared to other formats.
 * Does not require changing the source code of the class in any way.
```

Comment:
Can we change this sentence?
`Does not require a class to implement an interface or change the source code of the class in any way`

Reply:
done